vaadin-archetype-application/src/main/resources/archetype-resources/pom.xml

```diff
@@ -22,6 +21,7 @@
         <repository>
             <id>Vaadin Directory</id>
             <url>http://maven.vaadin.com/vaadin-addons</url>
+            <snapshots>false</snapshots>
```

Comment:
Does this work like this too? I have always use this:

            <snapshots>
                <enabled>false</enabled>
            </snapshots>

Reply:
It don't, tested. 
```suggestion
            <snapshots><enabled>false</enabled></snapshots>
```

Reply:
yeah. exactly the problem.

Reply:
I pushed a commit there ;-) i hope @pleku doesn't mind.

Reply:
You could have just pushed Commit suggestion button too ;-)